dart/lib/src/scope.dart

```diff
@@ -114,14 +126,14 @@ class Scope {
 
   final SentryOptions _options;
 
-  final List<SentryAttachment> _attachements = [];
+  final List<SentryAttachment> _attachments = [];
```

Comment:
Nice fix, since its a breaking change, should we keep the old one as an alias and add the @Deprecated annotation?